app/src/main/java/com/gdfp/android_evaluationv2/views/TaskTitleView.java

```diff
@@ -2,16 +2,18 @@
 
 import android.content.Context;
 import android.graphics.Paint;
-import android.support.v4.content.ContextCompat;
 import android.util.AttributeSet;
 
-import com.google.developer.taskmaker.R;
+import androidx.appcompat.widget.AppCompatTextView;
+import androidx.core.content.ContextCompat;
+
+import com.gdfp.android_evaluationv2.R;
 
 /**
  * Custom view to display the state of a task as well as
  * its description text.
  */
-public class TaskTitleView extends android.support.v7.widget.AppCompatTextView {
+public class TaskTitleView extends AppCompatTextView {
```

Comment:
I think this is the problem for the setText method that isn't working in the implementation above, please just keep last dependencies.